chain/consensus/mir/config.go

```diff
@@ -22,11 +23,24 @@ type BaseConfig struct {
 	CheckpointRepo string
 	// The name of the group of validators.
 	GroupName string
+	// The type of membership.
+	MembershipTypeValue string
 }
 
+const (
+	DefaultMembershipType = "file"
+	// ConfigOffset is the number of epochs by which to delay configuration changes.
+	// If a configuration is agreed upon in epoch e, it will take effect in epoch e + 1 + configOffset.
+	ConfigOffset    = 2
+	MaxProposeDelay = time.Duration(15)
```

Comment:
Let's do it the other way around, it is 1 by default for subnet and we set is explicitly to 15 for Spacenet mainnet. 
```suggestion
	MaxProposeDelay = time.Duration(1)
```